source/Locales/ScottLocale/ScottLocalesOfScottDomains.lagda

```diff
@@ -0,0 +1,816 @@
+---
+title:          The spectral Scott locale of a Scott domain
+author:         Ayberk Tosun
+date-started:   2023-10-25
+date-completed: 2023-11-26
+---
+
+In this module, we prove that the Scott locale of any Scott domain is a spectral
+locale (provided that the domain in consideration is large and locally small and
```

Comment:
I would remove "large and" since it's not essential that the domain is large, although it may be of course.

Reply:
If it's easy, then I would definitely generalize the universes so that it won't come back to bite you at some point in the future.

Reply:
I’ll try and see how easy it is.

Reply:
@tomdjong I tried to generalize the universes, but it there are a lot of dependencies that need to be generalized as well. In some of them, generalizing didn’t seem to be that easy. So I think we should merge this now and leave it as a future TODO to generalize.